e2e/cypress/integration/channel/channel_groups_spec.js

```diff
@@ -31,7 +31,9 @@ describe('channel groups', () => {
 
         cy.apiUpdateConfig({LdapSettings: {Enable: true}, ServiceSettings: {EnableTutorial: false}});
 
-        cy.apiLogin({username: 'board.one', password: 'Password1'});
+        cy.apiLogin({username: 'board.one', password: 'Password1'}).then(({user}) => {
+            cy.apiSaveCloudOnboardingPreference(user.id, 'hide', 'true');
+        });
```

Comment:
I was thinking if this could be put somewhere but can' figure out better location currently. So, let's leave it as is here until we needed a similar fix in the future.